app/controllers/sessions_controller.rb

```diff
@@ -1,28 +1,28 @@
 class SessionsController < ApplicationController
 
 
-  # login for registered users 
+  # login for registered users
   def create
     current_user = User.log_in(params[:session_data][:email], params[:session_data][:password])
     if current_user
       session[:user_id] = current_user.id
       @order = Order.create
-      @order.status = "pending" 
+      @order.status = "not_placed" 
       session[:order_id] = @order.id
       redirect_to products_path
     else
       redirect_to products_path
```

Comment:
Same comment as above, you can pull this out of the conditional since you call it anyway.